hardhat.config.ts

```diff
@@ -95,6 +98,18 @@ task('allowed-tokens', 'Retrieves a list of allowed tokens')
     await getAllowedTokens(hre);
   }
 );
+task('withdraw', 'Retrieves a list of allowed tokens')
```

Comment:
The description of the task is wrong

Reply:
Furthermore, I'd call it `collector:withdraw` since we're going to remove the script from `package.json`

Reply:
woops, my bad.

Reply:
done